### ISSUE TITLE
Support `GM_registerMenuCommand` and `GM_unregisterMenuCommand`

### ISSUE THREAD

**Issue by piquark6046** [AdguardTeam/CoreLibs#1698]
### Description
Currently, AdGuard does not support `GM_registerMenuCommand` and `GM_unregisterMenuCommand` that is required when an userscript developer wants to add its configuration feature, etc.

### Suggestion
 - Option button
   I think that placing the button here is better.
   <details><summary>Screenshot</summary>

   ![Screenshot from 2022-11-25 13-37-30](https://user-images.githubusercontent.com/98787049/203997412-ad7255f9-fca3-4477-ba6f-568803579641.png)

   </details>

- The `onClick` event must be executed only when any matched webpage is opened.

### Reference
 - https://violentmonkey.github.io/api/gm/#gm_registermenucommand

**Comment by ngorskikh** [comment]
I don't think we should give userscript developers an ability to execute code inside our applications. The applications don't even have a javascript engine to run it, anyway.
@ameshkov discuss?
AFAIK userscripts that want to have persistent configuration usually implement some sort of a menu on the page that they are injected into, and store/load the values with `GM_setValue`/`GM_getValue`.

**Comment by ameshkov** [comment]
Agreed. but I think we can implement this entirely in the page context without involving CoreLibs.

**Comment by bropines** [comment]
As far as I understand, it’s still hanging in todo?

**Comment by rakleed** [comment]
`GM.registerMenuCommand`, `GM.unregisterMenuCommand`

**Comment by AJolly** [comment]
I't'd be very useful, since right now you can't use AdGuard as your sole userscript manager.  

(and itd be great if you display a warning when an incompatible userscript is installed, did not realize this was the issue until I was digging through the console) "AdGuard Userscript Module] GM.registerMenuCommand is not supported"

**Comment by SeriousHoax** [comment]
Any update on this? Can we please get it with version 8 when it becomes stable?
